src/virt/csr.rs

```diff
@@ -213,6 +213,9 @@ impl RegisterContextGetter<Csr> for VirtContext {
             Csr::Vtype => self.csr.vtype,
             Csr::Vlenb => self.csr.vlenb,
 
+            // Crypto extension
+            Csr::Seed => 0x80000000,
```

Comment:
Which value do you need for the seed in the verification @francois141 ?

I see both `0x80000000` and `0x20000` in two different files?